contracts/SportMarkets/SportAMMRiskManager.sol

```diff
@@ -49,6 +50,7 @@ contract SportAMMRiskManager is Initializable, ProxyOwned, PausableUpgradeable,
 
     mapping(uint => uint) public dynamicLiquidityCutoffTimePerSport;
     mapping(uint => uint) public dynamicLiquidityCutoffDividerPerSport;
+    address public sportsAMMCancellationPool;
```

Comment:
I am not seeing it used anywhere in the contract, why is it needed here?

Reply:
it's used in SportsAMM, the only way the stack not to be too deep in `buyFromAMM()`